<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoDesktopClass</class>
 <widget class="QDialog" name="VideoDesktopClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoDesktop</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>176</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
background:white;
border-style:none;
border-radius:15px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="preview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>151</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#f3f3f3;
color:gray;
font: 10pt &quot;Microsoft YaHei UI&quot;;
border-radius:4px;</string>
    </property>
    <property name="text">
     <string>预 览 画 面</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getPath">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>…</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="loop">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>循环播放</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>视频地址</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enable">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>启用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>2</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Microsoft YaHei UI&quot;;
background:transparent;
border-style:none;</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="min">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>2</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
background:transparent;
border-style:none;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VideoDesktop.qrc"/>
 </resources>
 <connections/>
</ui>
